src/Tables/Table.tsx

```diff
@@ -1,33 +1,29 @@
-import React, {
-  FunctionComponent, ReactElement, useContext,
-} from 'react';
-import styled, { ThemeContext } from 'styled-components';
-import { BaseTheme } from '../Theme';
+import styled, { withTheme } from 'styled-components';
+import { BaseTheme } from 'Theme';
 import TableHead from './TableHead';
-import { TableBody } from './TableBody';
+import TableBody from './TableBody';
 
 export interface TableProps {
-  /** Internal table components like <thead> and <tbody> including their respective child components */
-  children: Array<TableHead | TableBody>;
+  /**
+  * Internal table components like <thead>, <tbody>, <col>. and <colgroup>
+  * including their respective child components
+  */
+  children: Array<typeof TableHead | typeof TableBody | HTMLTableColElement>;
```

Comment:
Hooooo boy. I added this because I was getting issues trying to use a value as a type, but going back and looking at it again it didn't really make much sense. This lead to a batch of updates to the typing for all of the Table components, which _should_ (I hope) fix the problem. Thanks for pointing that out.

Reply:
>  I added this because I was getting issues trying to use a value as a type

That sounds....painfully familiar.

> This lead to a batch of updates to the typing for all of the Table components, which should (I hope) fix the problem. Thanks for pointing that out.

Cool, thanks for clearing that up :)